### PR TITLE
Add getWithHelpWith

### PR DESCRIPTION
**Pull request by blackheaven**

### CODE REVIEW

File: src/Options/Generic.hs [hunk 1]
```diff
@@ -1133,8 +1134,20 @@ getWithHelp
     -- ^ Program description
     -> io (a, io ())
     -- ^ (options, io action to print help message)
-getWithHelp desc = do
-  a <- getRecordWith header mempty
+getWithHelp desc = getRecordWithHelp desc mempty
+
+-- | Marshal any value that implements `ParseRecord` from the commmand line
+-- alongside an io action that prints the help message.
+getRecordWithHelp
```
Comment (Gabriella439) [inline src/Options/Generic.hs]: For consistency, this should probably be called `getWithHelpWith`, even if that is kind of an awkward name

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,7 @@
+1.4.8
+
+* [Add `getRecordWithHelp`](https://github.com/Gabriel439/Haskell-Optparse-Generic-Library/pull/94)
```
Comment (Gabriella439) [inline CHANGELOG.md]: Don't forget to also change the `CHANGELOG` entry to match the new name

**Review by Gabriella439 — APPROVED** [review]

File: CHANGELOG.md (hunk 2 above)
Comment (blackheaven) [reply CHANGELOG.md]: Thanks, I have push forced for the commit name too

**Comment by Gabriella439** [comment]
Thank you for contributing this! 🙂

**Comment by blackheaven** [comment]
Thanks for your feedback

**Comment by Gabriella439** [comment]
You're welcome!

**Pull request merged**